### ISSUE TITLE
Add a connect peer endpoint

### ISSUE THREAD

**Issue by fusion44** [fusion44/blitz_api#92]
We need to connect to a peer anyway prior to opening a channel => Implement a `.../lightning/connect-peer` endpoint.

**Comment by rootzoll** [comment]
At the moment the open channel endpoint connects to the peer already before opening a channel. See: https://github.com/fusion44/blitz_api/blob/bb6d6a6bff3e47ba2408340c15f799e18e1e9e04/app/repositories/ln_impl/lnd.py#L438

**Comment by fusion44** [comment]
Yeah, my initial issue is worded badly. I do the same with the CLN implementation but figured this should be exposed as an API. This is just a reminder for later and is actually a very good first issue.
